shacl12-shacl-shacl/index.html

```diff
@@ -0,0 +1,415 @@
+				display: block;
+				font-family: sans-serif;
+				margin: -1em 0 0.4em -1em;
+				padding: 0.2em 1em;
+			}
+
+			pre.example-shapes {
+				background: #deb;
+			}
+
+			pre.example-shapes,
+			pre.example-shapes:before {
+				border: 1px solid #bbb;
+			}
+
+			pre.example-shapes:before {
+				color: #888;
+				content: "Example shapes graph";
+				width: 13em;
+			}
+
+			pre.example-data {
+				background: #eeb;
+			}
+
+			pre.example-data,
+			pre.example-data:before {
+				border: 1px solid #cc9;
+			}
+
+			pre.example-data:before {
+				color: #996;
+				content: "Example data graph";
+				width: 13em;
+			}
+
+			pre.example-results:before {
+				color: #797;
+				content: "Example validation results";
+				width: 13em;
+			}
+
+			pre.example-other {
+				background: #bed;
+			}
+
+			pre.example-other,
+			pre.example-other:before {
+				border: 1px solid #ddd;
+			}
+
+			pre.example-other:before {
+				color: #888;
+				content: "Example";
+				width: 13em;
+			}
+
+			pre.example-sparql {
+				background: #bed;
+			}
+
+			pre.example-sparql,
+			pre.example-sparql:before {
+				border: 1px solid #ddd;
+			}
+
+			pre.example-sparql:before {
+				color: #888;
+				content: "Example SPARQL";
+				width: 13em;
+			}
+
+			.example-results {
+				background: #edb;
+			}
+
+			.example-results,
+			.example-results:before,
+			.example-results th,
+			.example-results td {
+				border: 1px solid #cca;
+			}
+
+			/* our syntax menu for switching */
+			div.syntaxmenu {
+				border: 1px dotted black;
+				padding: 0.5em;
+				margin: 1em;
+			}
+
+			@media print {
+				div.syntaxmenu {
+					display: none;
+				}
+			}
+		</style>
+	</head>
+
+	<body>
+		<section id="abstract">
+			<p>This document describes the Shapes Constraint Language (SHACL) validator for SHACL data, referred to as "SHACL-SHACL".</p>
+			<p>
+				This specification is published by the <a href="https://www.w3.org/groups/wg/data-shapes/">Data Shapes Working Group</a>.
+			</p>
+		</section>
+
+		<section id="sotd"></section>
+
+		<section id="introduction">
+			<h2>Introduction</h2>
+			<p class="todo">TODO</p>
+		</section>
+
+		<section id="shapes">
+			<h2>Shapes</h2>
+			<p class="todo">TODO: This section will describe the Shapes created for SHACL-SHACL.</p>
+			<p>
+				These shapes are available in the RDF resource at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
+			</p>
+		</section>
+
+		<section id="governance">
+			<h2>Governance</h2>
+			<p class="todo">TODO: This section will describe the goverannce arrangements in place for the continued maintenance of this Note.</p>
+		</section>
+
+		<section id="security" class="appendix informative">
+			<h2>Security and Privacy Considerations</h2>
+			<p>
+				This Shapes graph does not import into or alter the data it is evaluating so will not affect an
+				organisation's information if applied to it, only report on its SAHCL Core validity. It will also not
```

Comment:
First, a nit: "SAHCL".

Bigger-picture question: Should SHACL-SHACL review all SHACL concepts, including those defined outside of Core?

Reply:
Resolved in https://github.com/w3c/data-shapes/pull/303/commits/19674920f126ab84bd2caed402eedded20ea9511